transforms/dialog-full-screen-app-wrapper/README.md

```diff
@@ -0,0 +1,26 @@
+# button-destructive
```

Comment:
Looks like you forgot to update this. L1-L3.

Reply:
Yep copy, paste and forget to update it 🤦